proto/monitor_service.proto

```diff
@@ -12,6 +12,7 @@ message StackTraceResponse {
   map<string, string> rpc_traces = 2;
   map<string, string> compaction_task_traces = 3;
   map<uint64, string> inflight_barrier_traces = 4;
+  repeated string barrier_worker_state = 5;
```

Comment:
Strictly speaking this is not a "stack trace" or "await tree". What about make it a field of `Diagnose` introduced by @zwang28 ?

Reply:
It seems that there is no struct named `Diagnose`. Diagnose is a http handler that collects and writes all types of diagnose information to the output string, and await tree output will be included in the output string via the `write_await_tree`.

Actually, I think we can extend the current rpc to a more general semantic, which is to collect diagnose information from all CNs. Previously it happened that all diagnose information is async stack trace, so the rpc is named stack trace. Now when we have more diagnose information, I think we can still reuse the rpc, and maybe just rename the rpc to avoid confusion.

When we want to fetch await tree information, we usually want to figure out the reason for barrier stuck, and it's harmless to include the barrier worker state in the output, which provides more information to help debug.